### ISSUE TITLE
Problems in training Camvid  dataset

### ISSUE THREAD

**Issue by Mayy1994** [jfzhuang/DAVSS#1]
Thanks for sharing so wonderful video semantic segmentation work.

I  did the experiments on Cityscapes dataset, and I can reproduce the data in your paper. However, I have encountered some problems when training the Camvid dataset. (The pretrained models are downloaded from the links you provided, and the requirements are satisfied.)

The PDA curves on testing are very weird, but the cfnet loss and the semantic loss are decreasing. Attached are the IoU accuracy across different testing epochs when the propagation distance is 1, 5, and 9, respectively. The last two images are the training losses.

![Screenshot from 2021-03-23 15-06-26](https://user-images.githubusercontent.com/40232175/112107199-9d70fa80-8be9-11eb-9a47-9562f7c98d7e.png)
![Screenshot from 2021-03-23 15-06-35](https://user-images.githubusercontent.com/40232175/112107202-9e099100-8be9-11eb-8b7d-b0be9ff1a54f.png)
![Screenshot from 2021-03-23 15-06-44](https://user-images.githubusercontent.com/40232175/112107204-9ea22780-8be9-11eb-8b83-e71be71cc645.png)

![Screenshot from 2021-03-23 15-06-55](https://user-images.githubusercontent.com/40232175/112107206-9f3abe00-8be9-11eb-80c5-5c75e6fe692e.png)
![Screenshot from 2021-03-23 15-07-06](https://user-images.githubusercontent.com/40232175/112107208-9fd35480-8be9-11eb-8b14-98f8d0ac1351.png)

I am confused about the above performances. The model seems over-fitting. Have you seen this kind of problem in your training? Do you have any idea how to fix this training problem?
In the "train.sh" file, both experiments adopt the same training epochs, learning rates, adjustment schedule of learning rates. Is there anything wrong with this setting?

Besides, I can obtain the exact same segmentation performances on Camvid just like the images in the repository folder "result --> spatial_correction_camvid" when I loaded the trained "spatial_correction_camvid" model provided in this repository. But unfortunately, I cannot achieve the same PDA in your paper. I can't figure it out.

<img width="425" alt="屏幕快照 2021-03-23 下午11 18 20" src="https://user-images.githubusercontent.com/40232175/112171925-3bd37f00-8c2f-11eb-862c-8c6e505ae4da.png">

          Fig. 12 in your paper

<img width="284" alt="Screenshot from 2021-03-23 23-19-02" src="https://user-images.githubusercontent.com/40232175/112172290-8c4adc80-8c2f-11eb-90d6-44a5f35a12c5.png">

          The PDA curve using the trained "spatial_correction_camvid" model provided in this repository.

Looking forward to your response. Many thanks.

**Comment by jfzhuang** [comment]
Many thanks for your interest in our work. I have to apologize that we have uploaded a previous version of codes and trained model weights for the camvid experiment by mistake. This version of codes lack of common data augmentation operations (such as random rotation and cropping), which are essential to alleviate the problem of overfitting. We will update the codes and corresponding model weights in the next few days. After that, You can conduct the camvid experiment again.

**Comment by jfzhuang** [comment]
We have updated the codes and trained model weight. Welcome to conduct the camvid experiment again.

**Comment by Mayy1994** [comment]
Thank you very much for your kind and quick response. I will conduct the experiment again.

**Comment by Mayy1994** [comment]
Hi, the updated "spatial_correction_camvid" model is the same as the previous one.
I have printed the parameters of this model, it is the same as the previous model. The PDA curve in the testing dataset also remains unchanged.
Could you please check whether you have uploaded the correct one?

Besides, I conducted the experiment on Camvid dataset (with the latest repository) again. I noticed that you added data augmentation of random cropping and random flipping, but the training becomes very unstable. The IoU accuracy across different testing epochs when the propagation distance is 1, 5, and 9 are shown in the images below. 

![Screenshot from 2021-03-26 20-22-50](https://user-images.githubusercontent.com/40232175/112632161-b7713e00-8e72-11eb-8914-a997bdb7e185.png)

I saved the best.pth and loaded it for testing. The PDA curve in the testing dataset using my trained model is shown in the following image. Still, I can't obtain the performance as your paper. It is similar to the previous performance.
![Figure_1的副本](https://user-images.githubusercontent.com/40232175/112638210-f951b280-8e79-11eb-90ed-a39f898fbc53.png)

**Comment by jfzhuang** [comment]
1. I'm so sorry to only update the model weight on the BaiduYun and forget about the Dropbox. Now, you can check the new model on the Dropbox.
2. Your training curve seems weird and you can check the curve in my camvid experiment in the following.
![camvid](https://user-images.githubusercontent.com/20696784/112714443-5c445780-8f15-11eb-8356-25b305037625.png)
Please check if you have updated the corresponding codes (e.g., dataset/camvid.py and python/train.py)

**Comment by Mayy1994** [comment]
Thank you so much! I have reproduced the experimental results on Camvid dataset. Bravo!
